packages/indexer-agent/src/agent.ts

```diff
@@ -11,6 +11,7 @@ import {
 import {
   Allocation,
   allocationRewardsPool,
+  RewardsPool,
```

Comment:
🤔 I don't see this used below. Is this import needed?

Reply:
no idea where it came from 🤨  rebasing and then will remove if still there

Reply:
I removed it and then found there's a reference to it at line 502 `uniqueRewardsPools: RewardsPool[]`. I don't think it belonged in this PR...

Reply:
Ok, now the changlist shows RewardsPool as removed.  Needs to be added back in and in the future if the PRs could be based off main and rebased on main before review that would make it easier to see what changes are made in this PR.